examples/amazonlinux2023.yaml

```diff
@@ -0,0 +1,14 @@
+# This template requires Lima v0.17.0 or later and requires macOS >=13.0 
+vmType: "vz"
```

Comment:
Why is the image incompatible with qemu?

Reply:
I misunderstood the documentation. Fixed.